hackathon_site/dashboard/frontend/src/components/cart/CartErrorBox/index.tsx

```diff
@@ -3,10 +3,25 @@ import { useSelector } from "react-redux";
 import { cartTotalSelector, errorSelector } from "slices/hardware/cartSlice";
 import Grid from "@material-ui/core/Grid";
 import { Alert, AlertTitle } from "@material-ui/lab";
+import { teamSizeSelector } from "../../../slices/event/teamSlice";
+import { maxTeamSize, minTeamSize } from "../../../constants";
```

Comment:
Absolute imports here too, since constants is in the root file it'll be `"constants.js"`